internal/grouper/grouper_models.go

```diff
@@ -26,6 +26,12 @@ func ConvertVulnerabilityToIDAliases(c []models.Vulnerability) []IDAliases {
 			idAliases.Aliases = append(idAliases.Aliases, v.Related...)
 		}
 
+		// For Ubuntu Security Advisory data,
+		// all related CVEs should be bundled together, as they are part of this USN.
+		if strings.Split(v.ID, "-")[0] == "USN" {
```

Comment:
hmm does this mean CVE in the related entries will show up as aliases in the output?

Reply:
nope, I think only `UBUNTU-CVE` will be shown in the group IDs (source CVEs shouldn't be present in the grouping results, as there's no Ubuntu data). And group IDs should only show one, which will be the USN. This is how we did it for DSA before.